src/__tests__/index.js

```diff
@@ -431,6 +431,12 @@ const unchanged = [
   [{opacity: () => {}}],
   [{objectPosition: 'center bottom'}],
   [{objectPosition: '5px 10px'}], // There's no RTL-flipped equivalent for the 5px. :-(
+  [{boxShadow: 'var(--shadow16)'}],
```

Comment:
with the exception of `boxShadow` the below tests were all passing before and after the changes in this PR